src/test/java/javaracingcarwinner/controller/JavaRacingGameWinnerTest.java

```diff
@@ -58,10 +59,17 @@ public class JavaRacingGameWinnerTest {
         cars.add(new RacingCar("loser", 1));
 
         List<RacingCar> winners = JavaRacingGame.whoIsWinners(cars);
+        List<String> names = winners.stream().map(w -> w.name()).collect(Collectors.toList());
 
-        assertThat(winners).hasSize(3);
+        assertAll(
+                () -> {
+                    assertThat(names).hasSize(3);
+                },
+                () -> {
+                    assertThat(names).contains("win", "win2", "win3");
+                }
+        );
```

Comment:
```suggestion
        assertAll(
                () -> assertThat(names).hasSize(3),
                () -> assertThat(names).contains("win", "win2", "win3")
        );
```
호출문이 하나이기 때문에 중괄호를 생략하여주셔도 되어요 :)